### PR TITLE
Completed html-tables

### PR DESCRIPTION
**Pull request by ayshahussain141**
Screenshot 2022-10-11 191419

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
```
Comment (Zhiwutian) [inline html-tables/index.html:8]: don't forget to add the `thead` and `tbody` elements in the proper places

File: html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <tr><th>Student Name</th><th>Course</th><th>Grade</th></tr>
```
Comment (Zhiwutian) [inline html-tables/index.html:9]: I would separate this line into multiple lines for clarity purposes

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates

**Comment by ayshahussain141** [comment]
Screenshot 2022-10-11 195618 2 o

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**